src/common/actions/repository-input.js

```diff
@@ -38,45 +34,14 @@ function checkStatus(response) {
   }
 }
 
-export function verifyGitHubRepository(repository) {
```

Comment:
`verifyGitHubRepository` was not used anymore, we do this on server-side when trying to create a snap.